clients/admin-ui/src/features/privacy-requests/RequestRow.tsx

```diff
@@ -140,7 +140,6 @@ const RequestRow: React.FC<{
     handleIdCopy,
     menuOpen,
     approveRequestResult,
-    denyRequestResult,
```

Comment:
hmm I think you may be able to remove this line as well, possibly? https://github.com/ethyca/fides/blob/bugfix_2367_required_denial_reasons/clients/admin-ui/src/features/privacy-requests/RequestRow.tsx#L48

Reply:
also maybe this line? https://github.com/ethyca/fides/blob/bugfix_2367_required_denial_reasons/clients/admin-ui/src/features/privacy-requests/RequestRow.tsx#L103

there's a lot going on in this file 😅

Reply:
Good catch. All of that can be removed. There is a lot going on there!